apisix/plugins/prometheus/exporter.lua

```diff
@@ -352,13 +360,50 @@ local function etcd_modify_index()
 end
 
 
+local function shared_dict_status()
+    local attr = plugin.plugin_attr(plugin_name)
+    if not attr or not attr.shared_DICT then
+        return
+    end
+
+    local name = {}
+    local set_shared_dict_param = function (shared_dict_name)
```

Comment:
We can put this in the for loop instead of defining a new function to solve it?
The function definition can't be JIT and should be avoided.

Reply:
> We can put this in the for loop instead of defining a new function to solve it? The function definition can't be JIT and should be avoided.

Would you provide doc about it? I would like to learn more about it.

Reply:
http://wiki.luajit.org/NYI#bytecode, see FNEW.

Reply:
> http://wiki.luajit.org/NYI#bytecode, see FNEW.

Thank you~